xml2rfc/run.py

```diff
@@ -315,6 +315,8 @@ def main():
                            help=configargparse.SUPPRESS),
     htmloptions.add_argument('--no-inline-version-info', action='store_false', dest="inline_version_info",
                            help=configargparse.SUPPRESS),
+    htmloptions.add_argument('--no-rfc-local', action='store_false', dest='rfc_local', default=True,
```

Comment:
Is setting this to `default=True` intensional?
This changes the existing behaviour by default.

Reply:
Sorry, ignore the above comment.
I got confused here. `action=store_false` doesn't change the default behaviour.